unit-tests/unit-test-default-main.cpp

```diff
@@ -0,0 +1,40 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+/*
+This file creates the default main for our unit-tests so that they can receive the 'context' option.
+If a test wants to define it's own main it must specify that using a cmake directive:
```

Comment:
`it's` == `it is`

Reply:
```
/*
This file is automatically added to any unit-test project generated by unit-test-config.py.

It provides the default main() for our unit-tests so that they can receive any custom options (such
as 'context').

If you don't want this file included, use this cmake directive:
    //#cmake:custom-main
Catch2 will then include its own main() by default.
If you want to create your own main(), define CATCH_CONFIG_RUNNER before including catch.h so Catch2
doesn't.
*/
```